recipes/sporco/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "sporco" %}
+{% set version = "0.1.10" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "13ff925142da922eb464f9124267521351a26c1bd039057f3eeecc8841430f0b" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  # Skip windows build since pyfftw dependency currently not available
+  # noarch: python
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - future
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.12
+    - scipy >=0.13.3
+    - imageio
+    - pytest
+    - pytest-runner
+  run:
+    - future
+    - python
+    - numpy >=1.12
```

Please pin compatible numpy.
```suggestion
    - {{ pin_compatible('numpy') }}
```

---

My understanding from the docs is that `pin_compatible('numpy')` is intended for packages that link against numpy, but this package just imports it as a module. Is `pin_compatible('numpy')` really appropriate here?

---

No, it is not appropriate then. Not pinning numpy will actually reduce the number of builds. Sorry for the confusion.